docs/docs/started/installing-nebari.md

```diff
@@ -0,0 +1,47 @@
+---
+id: installing-nebari
+---
+
+# Installing Nebari
+
+A guide to help you install Nebari for your project.
+
+This installation guide provides the basic needs to install and deploy Nebari for the first time and assumes the user is already familiar with the [Conda](https://docs.conda.io/projects/conda/en/latest/) and [Python packaging](https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-packages) ecosystems. If you are already familiar to Nebari and would like to check the advance configuration, feel free to skip to "advanced-settings" section.
+
+:::note
+This guide focus on installing Nebari for `Cloud` usage. For other instructions, please visit [deploying options](/started/deploy.md) for an overview of the available options and its respective installation steps.
+:::
+
+## Requirements
+
+Nebari heavily depends on [Terraform](https://www.terraform.io/) and a python ecosystem. The installation of the Terrraform binary is built in within Nebari source code and its is automatically downloaded during first execution. Currently, only `Linux` and `macOS` are supported, `Windows` only through the “Windows Subsystem for Linux” (see "WSL").
```

Comment:
```suggestion
Nebari heavily depends on [Terraform](https://www.terraform.io/) and a Python ecosystem. The installation of the Terraform binary is built in within the Nebari source code and it is automatically downloaded during the first execution. Currently, only `Linux` and `macOS` are supported.  `Windows` is only supported through the “Windows Subsystem for Linux” (see "WSL").
```

Reply:
We have in the readme that Windows is not supported. Should that change to WSL like you've mentioned here?

Reply:
well... we are actually creating a linux inside Windows, so I think we should keep it the way it is ... we can talk more on this next DevEx meeting

Reply:
Right, I meant to leave this as you have written and for me to go edit the readme so that it reads in a similar way. I'll make a note to discuss at our next meeting.